packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.mdx

```diff
@@ -53,6 +53,27 @@ switch (value) {
 
 Since `value` is a non-union type it requires the switch case to have a default clause only with `requireDefaultForNonUnion` enabled.
 
+### `considerDefaultExhaustiveForUnions`
+
+Defaults to false. Whether a `default` clause will be considered as making a switch statement over a union type exhaustive.
+
+If set to true, `switch` statement over a union type includes a `default` case is considered exhaustive.
+`considerDefaultExhaustiveForUnions` enforces explicitly handling every constituent of the union type with their own explicit `case`.
+This can be useful if you want to make sure every value added to the union receives explicit handling, with the `default` case reserved for reporting an error.
+
+Examples of additional **incorrect** code for this rule with `{ considerDefaultExhaustiveForUnions: true }`:
```

Comment:
```suggestion
Examples of additional **correct** code for this rule with `{ considerDefaultExhaustiveForUnions: true }`:
```

Reply:
I thought it would be good to provide detailed example cases, so I added a few cases each correct and incorrect.

Reply:
Makes sense 👍  I just went ahead and refactored this to our typical tabs setup